device-types/LiteOn/PF-2273-2L1M-LF.yaml

```diff
@@ -0,0 +1,87 @@
+---
+manufacturer: LiteOn
+model: PF-2273-2L1M-LF
+slug: liteon-pf-2273-2l1m-lf
+part_number: PF-2273-2L1M-LF
+u_height: 3
+is_full_depth: true
+airflow: front-to-rear
+subdevice_role: parent
```

Comment:
If all `device-bays` are removed, please remove this

Reply:
converted power supplies to modules, and removed related keys for parent/child devices